### PR TITLE
Powermon ports and other tweaks

### PR DESCRIPTION
**Pull request by jblance**

### CODE REVIEW

File: docker/powermon.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ commands:
     outputs:
       - type: mqtt
         format: 
-        - type: mqttmessage
+          type: mqttmessage
```
Comment (jblance) [inline docker/powermon.yaml:19]: this change is to match current config_model, question is should we allow multiple formats (in which case we'd want a list with a `-`) and need to change some logic

File: powermon/ports/__init__.py [hunk 2]
```diff
@@ -1,42 +1,66 @@
...
-    if portType is None:
+    if port_type is None:
         return None
 
     # build port object
-    if portType == PortType.SERIAL:
-        portObject = SerialPort.fromConfig(config=port_config)
-    elif portType == PortType.USB:
-        portObject = USBPort.fromConfig(config=port_config)
+    match port_type:
```
Comment (jblance) [inline powermon/ports/__init__.py:49]: I'm adopting the match pattern in preference to if, elif, else blocks where possible

**Review by rossandrews — APPROVED** [review]
Looks good, I am looking forward to trying out some bluetooth. I bought a Daly BMS with an active balancer I want to build a 24V lifepo4 battery with.

**Comment by jblance** [comment]
> Looks good, I am looking forward to trying out some bluetooth. I bought a Daly BMS with an active balancer I want to build a 24V lifepo4 battery with.

yeah I have a DALY as well (48V lifepo4 bank), ble needs a lot of work as the original was very specific to JKBMS

**Pull request merged**